src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -333,7 +333,6 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             // (The window has a min. size that ensures that there's always a scrollbar thumb.)
             if (drawableRange < 0)
             {
-                assert(false);

```

Comment:
this keeps showing up in your pull requests.. what is it

Reply:
context here: https://github.com/microsoft/terminal/pull/16809#discussion_r1511293498

since I've got marks in my prompt, they hit that assert literally every time I debug the Terminal. We concurred it's not really needed